apis/oathkeeper.json

```diff
@@ -2027,8 +2027,13 @@
         "match": {
           "$ref": "#/definitions/ruleMatch"
         },
-        "mutator": {
-          "$ref": "#/definitions/ruleHandler"
+        "mutators": {
```

Comment:
This should be updated automatically by the CI :) You can keep it as is - it will simply be overwritten once we merge the oathkeeper PR